### PR TITLE
Add zoning and occult zone

### PR DESCRIPTION
**Pull request by ajeanneney**
## Issue description :
- Get zoning and debatPublic metadata from sder database
- occult motivation if debat are not public

## Describe your changes :
- adding zoning and debatPublic to documentType
- creation of a new special categorie to occult motivation
- if decision debat are not public then create a special treatment to annotate motivations

## How to test :
- add zoning in storage or enable dbsder-api interraction to get TJ decisions with zoning directly from local sder database ( to enable dbsder-api : set `"db_api_enabled": true` in `localEnvironment.json` and use dbsder-api in the correct branch https://github.com/Cour-de-cassation/dbsder-api/tree/feat/call_zoning_on_create_decision)
- set debatPublic to false 
- check that motivation are well occulted after annotation
- try to find edge cases with overlapping annotations on motivations

## Checklist before requesting a review
- [x] I have performed a self-review of my code.
- [x] The feature works locally.
- [x] If it's relevant I added tests.
- [x] Will this be part of a product update? If yes, please write one phrase about this update.
> Occultation des motifs des décisions dont les débats ne sont pas publics. Cette occultation aura lieu dans une catégorie dédiée.

### CODE REVIEW

File: packages/generic/client/src/pages/Home/DocumentAnnotator/AnnotationsPanel/AnnotationsPanel.tsx [hunk 1]
```diff
@@ -164,6 +170,19 @@ function AnnotationsPanel(props: {
     );
   }
 
+  function renderPartiallyPublic() {
```
Comment (Bouba-cassation) [inline packages/generic/client/src/pages/Home/DocumentAnnotator/AnnotationsPanel/AnnotationsPanel.tsx]: Je ne sais pas si y'a un nom de fonction plus parlant :)

**Review by Bouba-cassation — APPROVED** [review]
C'est ok pour moi.

**Comment by ajeanneney** [comment]
@Bouba-cassation pour info : 
- rename du rendering
- throw error si l'annotation des motifs n'est pas possible
